recipe.yaml

```diff
@@ -40,5 +40,12 @@ Manifests:
                 -e SVCUID \
                 -v {kernel:rootPath}/ipc.socket:{kernel:rootPath}/ipc.socket \
                 greengrass_custom_secure_tunneling:NEXT_PATCH
+      Shutdown:
+        RequiresPrivilege: false
+        script: |-
```

Comment:
Why is this unprivileged? You're calling docker commands which would require you to be root or be in the docker group. The run lifecycle seems to assume that you would be root.

Reply:
I followed what was done in other labs component which uses docker (e.g: https://github.com/awslabs/aws-greengrass-labs-database-influxdb/blob/7ef7adc849ead5e47db8bbd7c97a58757f2e3229/recipe.yaml#L100 - the same apply for Grafana).

Reply:
@brnaba-aws since the `Run` commands use `RequiresPrivilege: true`, the newly introduced `Shutdown` commands should match that too.